codeMetadata.go

```diff
@@ -9,11 +9,14 @@ const (
 	MetadataPayable = 2
 	// MetadataReadable is the bit for readable flag
 	MetadataReadable = 4
+	// MetadataPayableBySC is the bit for payable flag
+	MetadataPayableBySC = 4
```

Comment:
This is confusing, we have to, at least, split them in 2 categories: the ones applied on the first byte and the ones applied on the second byte and so on.
At the first read I thought you just reused the value from the `MetadataReadable` . After further reading the whole code, I've found out that the `MetadataReadable` was applied on the first byte, not the second

Reply:
ok. split in too const groups